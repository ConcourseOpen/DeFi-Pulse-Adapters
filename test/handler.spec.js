const { handler } = require("../handler/handler");

describe("handler", () => {
  test("aave", async () => {
    const result = await handler({ queryStringParameters: { project: "fortube", timestamp: "0", block: "12202080" } });
    expect(result).toMatchSnapshot();
    console.log(result);
  }, 100000);
});
