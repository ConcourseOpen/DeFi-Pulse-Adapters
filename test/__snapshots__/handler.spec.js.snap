// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handler aave 1`] = `
Object {
  "body": "{\\"ETH\\":\\"693\\",\\"HBTC\\":\\"19\\",\\"WBTC\\":\\"0\\",\\"USDT\\":\\"768771\\",\\"USDC\\":\\"33771\\",\\"BUSD\\":\\"2525\\",\\"HUSD\\":\\"114617\\",\\"DAI\\":\\"46320\\",\\"FOR\\":\\"35932997\\",\\"HT\\":\\"33\\",\\"OKB\\":\\"52\\",\\"YFII\\":\\"0\\",\\"NEST\\":\\"73593\\",\\"ARPA\\":\\"34542\\",\\"YFI\\":\\"0\\",\\"LRC\\":\\"4\\",\\"SNX\\":\\"0\\",\\"HDOT\\":\\"4999\\",\\"HFIL\\":\\"12911\\",\\"QUSD\\":\\"180588\\",\\"BBTC\\":\\"0\\",\\"oBTC\\":\\"1\\"}",
  "headers": Object {
    "Content-Type": "application/json",
  },
  "statusCode": 200,
}
`;
